lamindb_setup/dev/_settings_storage.py

```diff
@@ -262,7 +262,7 @@ def type(self) -> str:
         import fsspec
 
         convert = {"file": "local"}
-        protocol = fsspec.utils.get_protocol(str(self.root))
+        protocol = fsspec.utils.get_protocol(self.root_as_str)
```

Comment:
It is called here

[lamindb-setup/lamindb_setup/dev/_hub_core.py](https://github.com/laminlabs/lamindb-setup/blob/3bff9bc24da7a9df7cdd00a84fe1599a5ce900ab/lamindb_setup/dev/_hub_core.py#L76)

Line 76 in [3bff9bc](https://github.com/laminlabs/lamindb-setup/commit/3bff9bc24da7a9df7cdd00a84fe1599a5ce900ab)

 type=ssettings.type, 
initializing the root and getting the credentials before the instance is registered in the hub.